src/oas/accessors.ts

```diff
@@ -25,3 +28,7 @@ const isValidOasParameter = (parameter: Partial<ParamTypeBase>): parameter is Pa
 export function getOasTags(tags: unknown): string[] {
   return Array.isArray(tags) ? tags.filter(tag => typeof tag !== 'object').map(String) : [];
 }
+
+export function getExtensions(target: Record<string, unknown>): Extensions {
+  return fromPairs(Object.entries(target).filter(([key]) => key.startsWith('x-') && !ROOT_EXTENSIONS.includes(key)));
```

Comment:
Why are we excluding `x-internal`?

Reply:
It is already being included separately as `internal` property in `HttpOperation`, so we don't want to double it